src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -470,7 +470,7 @@ void PolicyHandler::PushAppIdToPTUQueue(const uint32_t app_id) {
   LOG4CXX_AUTO_TRACE(logger_);
   sync_primitives::AutoLock lock(app_id_queue_lock_);
   const auto result = applications_ptu_queue_.insert(app_id);
-  if (result.second) {
+  if (result.second && PolicyEnabled()) {
```

Comment:
@YarikMamykin I think you don't need check `PolicyEnabled()` just only add `POLICY_LIB_CHECK_VOID();` instead and also maybe need add verification to `OnAddedNewApplicationToAppList` and `PopAppIdFromPTUQueue` functions?  Can you check this one?

Reply:
@ydementieiev I checked `PopAppIdFromPTUQueue` - this one executed only with enabled policies and `OnAddedNewApplicationToAppList` contains `PushAppIdToPTUQueue` only that is fixed by this PR.

Reply:
@ydementieiev POLICY_LIB_CHECK_VOID macro added in https://github.com/smartdevicelink/sdl_core/pull/3292/commits/0b561fa7c66d903bafc639f1908fea37ad5315d9

Reply:
@ydementieiev https://github.com/smartdevicelink/sdl_core/pull/3292/commits/c0ab36d8248e296f98c81b75500154f4cbfb332b